packages/lodestar/src/sync/regular/naive/naive.ts

```diff
@@ -61,6 +61,7 @@ export class NaiveRegularSync extends (EventEmitter as { new(): RegularSyncEvent
     const currentSlot = getCurrentSlot(this.config, state.genesisTime);
     if (headSlot >= currentSlot) {
       this.logger.info(`Regular Sync: node is up to date, headSlot=${headSlot}`);
+      this.emit("syncCompleted");
```

Comment:
should we also call stop here so it can remove chain listener and close source or sync calls that?

Reply:
`targetSlotRangeSource` and `controller` are not even initialized at that time

Reply:
but we registered chain listener which will get called nonetheless

Reply:
and targetSlotRangeSource is initialized in constructor

Reply:
oh didn't notice `targetSlotRangeSource` is initialized at both constructor and start()

so let's do everything in stop()

Reply:
have to emit `syncCompleted` separatedly because sync also call `stop()`